src/utils/helpers.ts

```diff
@@ -167,3 +167,38 @@ export const parseDate = (dateString: string): Date => {
   const [yyyy, mm, dd] = dateString.split('-').map((str) => parseInt(str));
   return new Date(yyyy, mm - 1, dd);
 };
+
+export const StringifyFormReplacer = (key: string, value: any): unknown => {
+  // an object with country is an address
+  if (
```

Comment:
Suggestion: Store the conditional in a constant, so that the check is more readable.

Example:

```
const isAddress = typeof value?.country !== 'undefined' && (!value.street || !value.city || (!value.postalCode && !value.zipcode));

if (isAddress) return undefined
```

I like this approach for when the check is long and not that straightforward to decipher.